### DISCUSSION TITLE
Custom Python scripts after compile or tests

### DISCUSSION THREAD

**Discussion by jvjmohan** [dbt-labs/dbt-core#5701] [Ideas]
Hi Team,

We have internal metrics to be gathered after a team builds a new model in dbt. 

Ideally, if any developer runs a compile or tests, at the end of it, Is there an option if I can invoke a custom python script? This custom python script will help us track the mertics. 

Any suggestions, please let me know.

**Comment by lostmygithubaccount** [comment]
Hi @jvjmohan, you might be interested in fal! I believe it serves this use case perfectly: https://github.com/fal-ai/fal
